### PR TITLE
El 1440 only run fala tests

### PR DESCRIPTION
**Pull request by starswan**
## What does this pull request do?

Only run FALA tests from end-to-end repo, so that FALA build times are reduced

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [x] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -160,6 +160,8 @@ jobs:
           source .circleci/define_build_environment_variables
           echo "export FALA_IMAGE=$ECR_DEPLOY_IMAGE" >> $BASH_ENV
           echo "export A11Y_ENABLED=true" >> $BASH_ENV
+          echo "export FALA_TESTS_ONLY=true" >> $BASH_ENV
+          echo "export E2E_BRANCH=EL-1440-add-conditional-fala-tests" >> $BASH_ENV
```
Comment (psweeting1) [inline .circleci/config.yml:164]: You've left `"export E2E_BRANCH=EL-1440-add-conditional-fala-tests" >> $BASH_ENV` In your PR.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: 2.1
 orbs:
   slack: circleci/slack@2.5.0
-  cla-end-to-end-tests: ministryofjustice/cla-end-to-end-tests@volatile
+  cla-end-to-end-tests: ministryofjustice/cla-end-to-end-tests@dev:alpha
```
Comment (psweeting1) [inline .circleci/config.yml:4]: Please can you revert back to `@volatile`: 

`@volatile` is a special tag that points to the most recent version of the orb.

File: .circleci/config.yml (hunk 1 above)
Comment (starswan) [reply .circleci/config.yml:164]: without it I can't prove that the tests run properly. I guess this should be in a separate branch or something...

**Review by psweeting1 — APPROVED** [review]
LGTM

**Pull request merged**